docs/introduction/motivation.md

```diff
@@ -0,0 +1,27 @@
+---
+id: motivation
+title: Motivation
+---
+
+Most applications have the logic for which they are created.
+This logic itself can be very complex and based on reactive principles, we call it **business logic**.
+
+In order for the user to benefit from the logic described above it is necessary to create a user interface
+and implement the logic for it, we call it **UI-logic**.
+
+The easiest way to figure out what is what — is to remember what tasks your project manager comes to you with,
+or what the product tasks look like. There's no description of how buttons, dropdowns, and form fields should work.
+There is a description of how the user interacts with the application.
+The users don't care what technologies and frameworks are used under the hood.
```

Comment:
I would say that there can be description of how buttons or dropdowns should work in product tasks - but this description will also be high-level and closer to users perspective.

Certainly, there will be no implementation details like "dropdown list state update should be React's low-priority transition" - it is up to the developer to decide, which frameworks and api's to use to achieve required level of quality.

I suggest to reflect this here too, something like that:
```suggestion
The easiest way to figure out what is what — is to remember what tasks your project manager comes to you with,
or what the product tasks look like.
There's no description of how buttons, dropdowns, and form fields should work internally or which api's should be used.
There is only a description of how the user interacts with the application.
The users don't care what technologies and frameworks are used under the hood - it is up to the developers to decide, which tools to use to achieve required quality.
```